rector.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
license header missing?

Reply:
Yeah you’re right. Should setup reuse in CI next.